content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,229 @@
+   :align: center
+   :alt: Show *Add to batch* window to create a batch transfer.
+
+Process batches
+===============
+
+To process batches, navigate to :menuselection:`Inventory app --> Operations --> Batch Transfers`.
+Click on a batch to select it.
+
+In the :guilabel:`Detailed Operations` tab, products that are to be picked are grouped by location.
+
+Under the :guilabel:`Source Package` or :guilabel:`Destination Package` field, enter the package
+used for the picking.
+
+.. note::
+   Use the :guilabel:`Source Package` field when the picking package is configured as *reusable* on
+   the :ref:`package form <inventory/misc/create-package>`. This means that the products are
+   temporarily placed in a container during picking before getting transferred to their final
+   shipping container.
+
+   Alternatively, use the :guilabel:`Destination Package` field when the product is directly placed
+   in its *disposable* shipping container during picking.
+
+.. example::
+   Process the cluster batch for the three orders of apples, oranges, and bananas :ref:`example
+   <inventory/misc/cluster_picking/example>` by assigning each picking to a dedicated package.
+
+   At the storage location for apples, `WH/Stock/Shelf A`, assign the apples in all three pickings
+   to one of the three disposable packages, `CLUSTER-PACK-1`, `CLUSTER-PACK-2`, and
+   `CLUSTER-PACK-3`.
+
+   Record this in Odoo using the :guilabel:`Destination Package` field in the
+   :guilabel:`Detailed Operations` tab.
+
+   .. image:: cluster_picking/cluster-batch-example.png
+      :align: center
+      :alt: Example of processing cluster pickings in *Inventory*
+
+In Barcode
+----------
+
+To process cluster pickings directly from *Barcode*, select the :guilabel:`Batch Transfers` button
+from the *Barcode* dashboard. Then, select the desired card for the batch.
+
+In the batch transfer screen, the products in the picking are grouped by location, and each line is
+color-coded to associate products in the same picking together.
+
+Then, follow the prompt to :guilabel:`Scan the source location` barcode for the storage location the
+first product is located at. Then, scan the barcode for the product and package to process the
+transfer. Repeat this for all products and click the :guilabel:`Validate` button.
+
+.. example::
+
+   .. list-table::
+      :header-rows: 1
+      :stub-columns: 1
+
+      * -
+        -
+      * - Begin processing the cluster picking by going to the first storage location, `Shelf A` and
+          scan the :ref:`location barcode <barcode/setup/location>`. Doing so highlights all the
+          pickings that need products from this particular location.
+
+          Scan the barcode for the apple, which highlights the picking labeled in red for the product
+          `Apple` at for the picking, `WH/PICK/00007`.
+
+          Then, scan the `CLUSTER-PACK-1` package barcode, and place the product in the designated
+          package.
```

Comment:
Hello @samueljlieber! Do you have suggestions on how to [display text next to an image](https://runbot176.odoo.com/runbot/static/build/50449452-16-0/logs/build/html/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.html#process-batches) (just to conserve space because i thought the block was a little long)? I put the text in a table, but the text is currently bolded.

From a technical standpoint, I wanted to ask if displaying the text next to the image is even a good idea, because maybe it would render wonky on a smaller screen?

Reply:
Hi @Felicious I'd avoid putting an image aside text because it will not wrap below the text on smaller screens, making both the text and image difficult to read. 

I suggest keeping the format of your other example blocks where the text comes first and the image is below. I also suggest reducing the width of the image to 388px, this will help keep it from appearing so large on desktop. 🙂